packages/kbn-i18n/src/loader.test.js

```diff
@@ -0,0 +1,277 @@
+/*
+ * Licensed to Elasticsearch B.V. under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch B.V. licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import { join } from 'path';
+
+describe('I18n loader', () => {
+  let i18nLoader;
+
+  beforeEach(() => {
+    jest.resetModules();
+    i18nLoader = require('./loader');
+  });
+
+  describe('registerTranslationFile', () => {
+    test('should throw error if path to translation file is not specified', () => {
+      expect(() => i18nLoader.registerTranslationFile()).toThrow();
```

Comment:
__question:__ I tend to think that `toThrowErrorMatchingSnapshot` would be really useful here as well, we should make sure we generate appropriate message where possible since it's a part of API that i18n package exposes, what do you think?